### ISSUE TITLE
Getting NaN backtest results for DDPG and TD3 algorithm for a single stock

### ISSUE THREAD

**Issue by Athe-kunal** [AI4Finance-LLC/FinRL#258]
I tried out few individual stocks like TSLA, GOOGL and AMZN. But I am encountering NaN backtest results for the same
![image](https://user-images.githubusercontent.com/40949756/128005909-70e037c1-6302-42d4-af1c-28f5b78cb384.png)

Going through the results while training, I found this 
![image](https://user-images.githubusercontent.com/40949756/128006036-17c6f17e-5cad-4891-8c38-436c3495579f.png)

Here the actor loss is critically high. Am I getting something wrong? I trained it for GOOGL, but I didn't get this issue with TSLA or DOW Jones 30. Also, I tried a bunch of other stocks like AAPL, AMZN etc. I got these NaN results because the final and initial portfolio value is same.

**Comment by rayrui312** [comment]
I tried TSLA using DDPG but didn't find the same issue. Could you please give your specific codes so that we can help figure it out?

**Comment by Athe-kunal** [comment]
Yeah after re-running the experiments for individual stock 'AAPL', DDPG gave results but TD3 gave NaN values, so probably tuning the learning rate will ameliorate this issue. Thank you

**Comment by rayrui312** [comment]
Yeah. Basically, it is hard to get good results for different data using constant parameters. We are further tuning the hyperparameters now.

**Comment by Athe-kunal** [comment]
Hmm, it surely does require a lot of training. Also, if we add a try/except block for the Sharpe ratio, then we can get rid of NaN values and ZeroDivisionError while training. So like 
try:
  sharpe = .....
except:
  sharpe = 0.0
In my testing, I was zero division error while training, probably we can include this
